src/services/workspace-client/CheWorkspaceClient.ts

```diff
@@ -47,16 +49,16 @@ export class CheWorkspaceClient {
         }
       };
       updateTimer();
-      axios.interceptors.request.use(async request => {
+      this.axios.interceptors.request.use(async request => {
         const keycloak = KeycloakSetup.keycloakAuth.keycloak as any;
         if (keycloak && keycloak.updateToken && !isUpdated) {
           updateTimer();
           try {
             await new Promise((resolve, reject) => {
               keycloak.updateToken(5).success((refreshed: boolean) => {
-                if (refreshed && keycloak.token && axios.defaults && axios.defaults.headers && axios.defaults.headers.common) {
+                if (refreshed && keycloak.token && this.axios.defaults && this.axios.defaults.headers && this.axios.defaults.headers.common) {
```

Comment:
I'd use optional chaining here and set authorization header if it's provided in any case:
```suggestion
                if (refreshed && keycloak.token) {
                  if(this.axios.defaults?.headers?.common === undefined) {
                    this.axios.defaults = { headers: { common: {} };
                  }
```

Reply:
I have fixed